src/client.js

```diff
@@ -2,17 +2,19 @@
 class Client {
   constructor(url) {
     this.url = url;
+    this.listLocations = this.listLocations.bind(this);
   }
 
   createConnectionToken() {
     const formData = new URLSearchParams();
     return this.doPost(this.url + "/connection_token", formData);
   }
 
-  registerDevice({ label, registrationCode }) {
+  registerDevice({ label, registrationCode, location }) {
     const formData = new URLSearchParams();
     formData.append("label", label);
     formData.append("registration_code", registrationCode);
+    formData.append("location", location);
```

Comment:
what happens when we append `null` for location? Even with Mandatory Locations we might have users who don't have any locations and we set `registerLocationId` to [null](https://github.com/stripe/stripe-terminal-js-demo/pull/89/files#diff-30658374b003f4b730d0803706718e81fe2938a3e4fd54e28fb590135a7c969aR29) for those cases. Do we know if backend can handle null values?